doc/fuzzing.md

```diff
@@ -198,6 +198,7 @@ $ cmake -B build_fuzz \
    -DCMAKE_C_COMPILER="$(pwd)/honggfuzz/hfuzz_cc/hfuzz-clang" \
    -DCMAKE_CXX_COMPILER="$(pwd)/honggfuzz/hfuzz_cc/hfuzz-clang++" \
    -DBUILD_FOR_FUZZING=ON \
+   -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION=ON \
```

Comment:
This won't define the c++ macro `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`, as this only sets a cmake build flag (cmake probably complains that this flag doesn't exist).

You should be able to use `-DAPPEND_CPPFLAGS="-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION"` (I see where the confusion comes from, as `-D` is accepted by compilers to define macros and by cmake to set build flags but they are different things). 

You will also need to add this to the correct section in the docs (i.e. the netdriver docs):

https://github.com/bitcoin/bitcoin/blob/ffdbf6c45be2a69050401cd5dad785758cee21d4/doc/fuzzing.md?plain=1#L246-L258

Reply:
> You should be able to use -DAPPEND_CPPFLAGS="-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION" (I see where the confusion comes from, as -D is accepted by compilers to define macros and by cmake to set build flags but they are different things).

Ah yes, thank you. Just saw some examples from the CI scripts.